python/cucim/src/cucim/skimage/measure/_colocalization.py

```diff
@@ -261,7 +255,8 @@ def manders_overlap_coeff(image0, image1, mask=None):
     if image1.min() < 0:
         raise ValueError("image1 contains negative values")
 
-    return _get_manders_overlap_coeff(image0, image1)
+    denom = cp.linalg.norm(image0) * cp.linalg.norm(image1)
+    return cp.vdot(image0, image1) / denom
```

Comment:
These changes are all in PR ( https://github.com/rapidsai/cucim/pull/578 )

Basically just trying to push as much of the work into cuBLAS as possible to make fusion a non-issue. Also seemed better to inline once done